draft-ietf-webtrans-http2.md

```diff
@@ -197,7 +197,7 @@ the site that requested the creation of the session.
 Upon receiving an extended CONNECT request with a `:protocol` field set to
 `webtransport`, the HTTP server checks if the identified resource supports
 WebTransport sessions. If the resource does not, the server SHOULD reply with
-status code 404 ({{Section 6.5.4 of !RFC7231}}). If it does, it MAY accept the
+status code 406 ({{Section 6.5.6 of !RFC7231}}). If it does, it MAY accept the
```

Comment:
nit: please refer to 9110

Reply:
Fixed in [c6cea30](https://github.com/ietf-wg-webtrans/draft-webtransport-http2/pull/65/commits/c6cea30123e9e098ad42530faa02cf28f31c0a9a)